conformance/06_tags_list_test.go

```diff
@@ -0,0 +1,72 @@
+package conformance
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/bloodorangeio/reggie"
+	g "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var test06TagsList = func() {
+	g.Context("Tags List", func() {
+		g.Specify("GET request to /v2/<name>/tags/list should yield 200 response", func() {
```

Comment:
"GET request to /v2/<name>/tags/list" -> "Get request to list tags"

Reply:
done